homeassistant/components/google_assistant/trait.py

```diff
@@ -1049,7 +1050,9 @@ def query_attributes(self):
             # Google will not issue an open command if the assumed state is
             # open, even if that is currently incorrect.
             if self.state.attributes.get(ATTR_ASSUMED_STATE):
-                response['openPercent'] = 50
+                response['willReportState'] = False
```

Comment:
This is wrong. reportState is a sync attribute and we already set it to False. You need to raise an error instead

```
            raise SmartHomeError(
                ERR_FUNCTION_NOT_SUPPORTED,
                'Querying state is not supported')
```

Reply:
And you need to raise that error when it's both assumed state and when it's unknown.

Reply:
Ok, but it won't fix the issue. It looks like google assistant is making a query before executing an open/close command. When we raise an error on query it raises the same error while executing the command. While CommandOnly is not available can we just assume openPercent = 50? It would allow us to open and close stateless covers.

Reply:
I wasn't raising the right error. Now with ERR_NOT_SUPPORTED it works. Stateless covers are moving but google assistant says otherwise due to the raised error.